### ISSUE TITLE
Course Feedback

### ISSUE THREAD

**Issue by SalesforceLynx** [crossworth/the-confident-spring-professional#1]
Hello Pedro, 

Hope you're doing great. I guess you had the course of Marco Behler. I want to know if that was helpful and you recommend it to someone who want to learn spring framework? Please share your experience. Thanks.

Best Regards,
Lynx

**Comment by crossworth** [comment]
Hi @SalesforceLynx, sorry for the late reply.

Yes, I took the Marco Behler course, at the time I trying to get a easy to understand course about spring, my knowledge at the time was very shallow, having never done anything with spring.

I liked the course, it does not go in deep how things works under the hood, but gives a complete overview of the ecosystem, it start slow, showing how things can be done without spring and improving each time using more and more the framework.

Since I was just starting with the framework, for me was great, now if you have some knowledge how spring works, you may not have the same experience as me.

In summary, it's a great introduction for the spring ecosystem, it's relatively short (in hours) so you can start and finish in a weekend, it should not be compared with a "From zero to hero" course. You probably will have to find other courses/material to complement your knowledge.

Feel free to ask more concrete questions.

I hope this help you in some way, have a nice day.

**Comment by SalesforceLynx** [comment]
Thanks Pedro, for the reply. I'm following the Laur Spilca book Spring start here. I find it quite easy to understand and you can think of my experience as similar to yours back then.

I appreciate the feedback you provided but I'm now looking for some more concrete small applications which covers all(major) aspects of the framework. So that I can train my mind to think like a Spring developer.

If you have came across such repos or YouTube videos please share. Thanks.
